src/utils/webrtc/models/CallParticipantModel.js

```diff
@@ -240,6 +242,12 @@ CallParticipantModel.prototype = {
 		}
 	},
 
+	_handleRaisedHand: function(data) {
+		// FIXME: rely on peer id ?
```

Comment:
@danxuliu should we check the peer id like for other methods ?

As far as I understood a peer id is only available if there's at least an audio or video stream ?